.github/workflows/release.yml

```diff
@@ -1,5 +1,7 @@
 name: kwctl release
 on:
+  # User can trigger this workflow in the Github UI
+  workflow_dispatch:
```

Comment:
for other workflows this is useful, but for the release one, I would prefer not to have it so releases are released once and never overwritten.